ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -698,16 +690,17 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
       case class Reject(_operation: String, userId: String)(implicit
           loggingContext: ContextualizedErrorLogger
       ) extends LoggingTransactionErrorImpl(
-            cause = s"cannot ${_operation} for unknown user \"${userId}\"."
-            // TODO (i12053): also output participantId
+            cause = s"cannot ${_operation} for unknown user \"${userId}\""
           ) {
         override def resources: Seq[(ErrorResource, String)] = Seq(
           ErrorResource.User -> userId
         )
       }
     }
-    @Explanation("There already exists another user with the same user-id.")
-    @Resolution("Choose a different user-id or use the user that already exists.")
+    @Explanation("There already exists a user with the same user-id.")
+    @Resolution(
+      "Check that you are connecting to the right participant node and the user-id is spelled correctly and/or use the user that already exists."
```

Comment:
and/or is rarely a good choice of wording. Often it confuses the readers as it is not clear whether you mean 'and' colloquial English, 'or' in mathematical sense or 'xor' in a legalese

Reply:
fixed in a new commit